src/core/tests/type_prop/is_finite.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+#include "openvino/opsets/opset10.hpp"
+#include "util/type_prop.hpp"
+
+using namespace std;
+using namespace ov::opset10;
+
+TEST(type_prop, isfinite_output_shape) {
+    const auto data = make_shared<Parameter>(ov::element::f16, ov::Shape{4, 2});
+    const auto isfinite = make_shared<IsFinite>(data);
+
+    EXPECT_EQ(isfinite->get_output_partial_shape(0), ov::PartialShape({4, 2}))
+        << "The output shape of IsFinite is incorrect";
+    ASSERT_EQ(isfinite->get_shape(), (ov::Shape{4, 2})) << "The output shape of IsFinite is incorrect";
+}
+
+TEST(type_prop, isfnite_sample_dynamic_batch) {
```

Comment:
Interval dimensions should be tested too, please take a look at the (https://github.com/openvinotoolkit/openvino/pull/13369#discussion_r991533375)